tasks/7_random-fraction/random-fraction.js

```diff
@@ -0,0 +1,3 @@
+export function randomFraction() {
```

Comment:
There is the test in `tasks/7_random-fraction/random-fraction.js`

`
    test('should return random decimal number', () => {
        const n0 = randomFraction();

        expect(randomFraction()).not.toBe(n0);
    });
`

In order to pass this test we have to be sure that `Math.random` won't generate same number twice in a row.

Reply:
Fixed.